src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -417,4 +420,4 @@
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
nit: I think you accidentally removed the new line at the end (VS has a habit of doing that on your behalf when editing csproj files)